python/lsst/cp/pipe/cpCertify.py

```diff
@@ -69,19 +75,78 @@ def __init__(self, *, registry, inputCollection, outputCollection, lastRunOnly=T
         self.inputCollection = inputCollection
         self.outputCollection = outputCollection
 
-    def run(self, datasetTypeName, timespan):
+    def run(self, datasetTypeName, beginDate=None, endDate=None):
         """Certify all of the datasets of the given type in the input
         collection.
 
         Parameters
         ----------
         datasetTypeName : `str`
             Name of the dataset type to certify.
-        timespan : `lsst.daf.butler.Timespan`
-            Timespan for the validity range.
+        beginDate : `str`
+            Date this calibration should start being used.
+        endDate : `str`
+            Date this calibration should stop being used.
         """
+        timespan = Timespan(
+            begin=astropy.time.Time(beginDate) if beginDate is not None else None,
+            end=astropy.time.Time(endDate) if endDate is not None else None,
+        )
+
         refs = set(self.registry.queryDatasets(datasetTypeName, collections=[self.inputCollection]))
         if not refs:
             raise RuntimeError(f"No inputs found for dataset {datasetTypeName} in {self.inputCollection}.")
         self.registry.registerCollection(self.outputCollection, type=CollectionType.CALIBRATION)
         self.registry.certify(self.outputCollection, refs, timespan)
+
+
+def build_parser():
+    """Construct the argument parser for ``certifyCalibrations.py``.
+
+    Returns
+    -------
+    parser : `argparse.ArgumentParser`
+        The constructed argument parser.
+    """
+
+    parser = argparse.ArgumentParser(description="""
+    Certify a cp_pipe generated calibration as valid for a given date
+    range, and register them with the butler.
+    """)
+
+    parser.add_argument("root", help="Path to butler to use")
+    parser.add_argument("inputCollection", help="Input collection to pull from.")
+    parser.add_argument("outputCollection", help="Output collection to add to.")
+    parser.add_argument("datasetTypeName", help="Dataset type to bless.")
+
+    parser.add_argument("-b", "--beginDate",
+                        help="Start date for using the calibration")
```

Comment:
What is the format of this date string? ISO8601? TAI vs UTC? Can it be MJD?  If you don't specify it what happens?